angular-workspace/projects/ni/nimble-angular/src/thirdparty/README.md

```diff
@@ -3,8 +3,9 @@ This folder contains forked third-party source.
 - Each file in this folder should begin with a comment explaining where the file comes from and listing any modifications from the original.
 - All comments made by Nimble developers should be prefixed with the tag "`[Nimble]`".
 
-The `directives` folder corresponds to the [`packages/forms/src/directives` folder](https://github.com/angular/angular/tree/14.2.6/packages/forms/src/directives) of the Angular repo. **When updating the version of Angular** used by the `nimble-angular` package:
+The `directives` folder corresponds to the [`packages/forms/src/directives` folder](https://github.com/angular/angular/tree/15.2.0/packages/forms/src/directives) of the Angular repo. **When updating the version of Angular** used by the `nimble-angular` package:
```

Comment:
It looks like the latest 15.2 tag is actually 15.2.10: https://github.com/angular/angular/blob/15.2.10/packages/forms/src/directives/number_value_accessor.ts

Thoughts on validating against the latest changes in 15.2? Looking briefly at the changes from 15.2.0 -> 15.2.10 it doesn't look like many of the files we copied are updated but may be worth double checking and updating these references: https://github.com/angular/angular/compare/15.2.0...15.2.10

We also may want to point out using the GitHub compare feature as a hint during upgrades to see potential changes easier.

Reply:
None of our forked files changed between 15.2.0 and 15.2.10, so I updated all the links at the tops of those files. I also updated the README to suggest using GitHub compare to do updates. I also updated the peer dependency versions in nimble-angular's package file.

Reply:
@m-akinc, why did the peer dependency versions need to change? Is there a requirement that clients can only use 15.2.10 or higher of Angular?

Reply:
We've decided to stick with 15.2.10.